fbs/matrix.fbs

```diff
@@ -22,39 +22,78 @@
 
 namespace NetEncoding;
 
-table Float32Array {
+table Float32FBArray {
```

Comment:
the addition of `FB` to clarify that this is a FB type is good -- adds clarity, but I suggest you do it uniformly to all types defined in this file.

it also might be clearer if it is a prefix, eg, FBFloat32Array, FBColumn, etc. (I don't feel strongly about this)